javalib/src/main/scala/java/lang/Integer.scala

```diff
@@ -373,16 +383,19 @@ object Integer {
   @inline def rotateRight(i: scala.Int, distance: scala.Int): scala.Int =
     (i >>> distance) | (i << -distance)
 
-  @inline def signum(i: scala.Int): scala.Int =
-    if (i == 0) 0 else if (i < 0) -1 else 1
+  @inline def signum(i: scala.Int): scala.Int = {
+    // Hacker's Delight, Section 2-7
```

Comment:
```suggestion
    // Hacker's Delight, Section 2-8
```

? (I'm looking at second edition: https://doc.lagout.org/security/Hackers%20Delight.pdf)